### PR TITLE
feat(dispatch,ontology): add terms for GINsim format

### PR DESCRIPTION
**Pull request by jonrkarr**
- Added EDAM terms for GINsim formats
- Added example simulation runs for GINsim, LibSBMLsim
- Debugged issues with OpenAPI spec for the COMBINE service

### CODE REVIEW

File: apps/combine-service/src/spec/spec.yml [hunk 1]
```diff
@@ -417,6 +417,9 @@ paths:
                       master: true
                   files:
                   - <FILE>
+            encoding:
```
Comment (jonrkarr) [inline apps/combine-service/src/spec/spec.yml:420]: @bilalshaikh42 in my experience, this needed to be added back. Please review.

Comment (bilalshaikh42) [reply apps/combine-service/src/spec/spec.yml:420]: I was having some trouble with generating a client library with that field present. What is the reason it is needed? I am not familiar with the function of the "encoding" field.

Comment (bilalshaikh42) [reply apps/combine-service/src/spec/spec.yml:420]: I see the issue. i did not realize that the encoding was needed for the input validation.

**Review by bilalshaikh42 — APPROVED** [review]

**Comment by bilalshaikh42** [comment]
Merged

**Pull request closed without merging**